### ISSUE TITLE
TypeError: Cannot read property 'namespaceURI' of undefined

### ISSUE THREAD

**Issue by IanDavey** [Xcential-Corporation/XSLTjs#5]
Following from #4, I get `TypeError: Cannot read property 'namespaceURI' of undefined`.  Is namespace-handling implemented?

**Comment by grantcv1** [comment]
Yes, namespace handling is implemented. I need more details, however.
